src/material/progress-spinner/progress-spinner.ts

```diff
@@ -264,16 +274,22 @@ export class MatProgressSpinner extends _MatProgressSpinnerMixinBase implements
     return this.strokeWidth / this.diameter * 100;
   }
 
+  /** Returns a string that is used for setting the spinner animation-name CSS property */
+  get _spinnerAnimationLabel() {
+    // Period '.' character cannot be included in an animation-name
+    return this._circleDiameter.toString().replace('.', '_');
```

Comment:
This will be called on every change detection run, but it only changes when the diameter is changed. It might be better if we used a regular property that gets updated once per input change.

Reply:
Yes, missed this one. Updated the PR with the corrections.